esp/src/eclwatch/nls/es/hpcc.js

```diff
@@ -555,17 +574,17 @@ define(
     RemoteDaliIP: "Remote&nbsp;Dali&nbsp;IP&nbsp;Address",
     Remove: "Remover",
     RemoveAttributeQ: "Está por eliminar este atributo. Está seguro de que desea continuar?",
-    RemovePart: "Eliminar parte",
     RemoveAtttributes: "Eliminar atributo(s)",
-    RemoveSubfiles2: "Remover subarchivo(s)",
+    RemovePart: "Eliminar parte",
     RemoveSubfiles: "Borrar Sub-Archivo(s)",
+    RemoveSubfiles2: "Remover subarchivo(s)",
     RemoveUser: "Esta apunto de excluirse del grupo:",
     Rename: "Renombrar",
     RenderedSVG: "SVG creado",
     RenderSVG: "Crear SVG",
     Replicate: "Replicar",
     ReplicatedLost: "Réplica perdida",
-    ReplicateOffset: "Replicate Offset",
+    ReplicateOffset: "Replicar Offset",
```

Comment:
we need more context for this messages, but as is, it sounds like a request to replicate an offset...
when in fact we might be referring to an offset of a replicate??
in that case it would be "offset de reproducción"